src/module/item/base/sheet/sheet.ts

```diff
@@ -499,6 +502,14 @@ class ItemSheetPF2e<TItem extends ItemPF2e> extends ItemSheet<TItem, ItemSheetOp
             });
         }
 
+        // Handle contenteditable fields
+        for (const input of htmlQueryAll<HTMLSpanElement>(html, "span[contenteditable][data-property]")) {
```

Comment:
If you have to assert anyway, just use `html.querySelectorAll`

Reply:
Looks like the assert (is that actually called an assert? Seems very different from the usual cases I see) isn't necessary. Maybe it was necessary in a previous version of the thing and it carried on through.

Reply:
It's called a [type assertion](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions), but maybe the DOM lib is wise to `${tagName}[${string}]`.